src/auth-service/utils/control-access.js

```diff
@@ -121,10 +122,13 @@ const controlAccess = {
       const { user_id, token } = params;
       const limit = parseInt(request.query.limit, 0);
       const skip = parseInt(request.query.skip, 0);
-
+      const timeZone = moment.tz.guess();
       let filter = {
         token,
         user_id,
+        expires: {
+          $gt: moment().tz(timeZone).toDate(),
+        },
```

Comment:
@Codebmk / @123MwanjeMike 
checks for the validity of the `access token`. This default validity is added in the access token creation via `config.constants`:

```
1.  EMAIL_VERIFICATION_HOURS: 1,
2.  EMAIL_VERIFICATION_MIN: 0,
3.  EMAIL_VERIFICATION_SEC: 0,
```